### PR TITLE
[PRM-540]: added non-VAT penalty reasons

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

File: test/models/ETMPPayloadSpec.scala [hunk 1]
```diff
@@ -257,7 +257,7 @@ class ETMPPayloadSpec extends AnyWordSpec with Matchers {
       | "latePaymentPenalties": [
       |     {
       |       "type": "additional",
-      |       "reason": "VAT_NOT_PAID_AFTER_30_DAYS",
```
Comment (ruthdas03) [inline test/models/ETMPPayloadSpec.scala:260]: Not sure why the reason "VAT_NOT_PAID_AFTER_30_DAYS" was replaced by "OFFICERS_ASSESSMENT_NOT_PAID_WITHIN_15_DAYS"

File: app/models/reason/PaymentPenaltyReasonEnum.scala [hunk 2]
```diff
@@ -34,6 +43,15 @@ object PaymentPenaltyReasonEnum   extends Enumeration {
         case "VAT_NOT_PAID_WITHIN_15_DAYS" => JsSuccess(VAT_NOT_PAID_WITHIN_15_DAYS)
         case "VAT_NOT_PAID_WITHIN_30_DAYS" => JsSuccess(VAT_NOT_PAID_WITHIN_30_DAYS)
         case "VAT_NOT_PAID_AFTER_30_DAYS"  => JsSuccess(VAT_NOT_PAID_AFTER_30_DAYS)
```
Comment (ruthdas03) [inline app/models/reason/PaymentPenaltyReasonEnum.scala:45]: Scala checker complains 'Cyclomatic complexity of 14 exceeds max of 10. ' against these lines of case . Are we sure we can ignore it.

**Review by ruthdas03 — COMMENTED** [review]
The changes are fine, except for couple of minor queries .

File: test/models/ETMPPayloadSpec.scala (hunk 1 above)
Comment (BackRowOla) [reply test/models/ETMPPayloadSpec.scala:260]: The change wasn't needed. I forgot to revert this back once I had made the changes to the ReasonsEnum.

File: app/models/reason/PaymentPenaltyReasonEnum.scala (hunk 2 above)
Comment (BackRowOla) [reply app/models/reason/PaymentPenaltyReasonEnum.scala:45]: The team has agreed to ignore complexity warning for this specific situation

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**